lms/djangoapps/support/static/support/js/collections/certificate.js

```diff
@@ -0,0 +1,22 @@
+;(function (define) {
+    'use strict';
+    define(['backbone', 'support/js/models/certificate'],
+        function(Backbone, CertModel) {
+            var CertCollection = Backbone.Collection.extend({
+                model: CertModel,
+
+                initialize: function(options) {
+                    this.userQuery = options.userQuery || "";
```

Comment:
Nitpick: consistency of quotes (single used on line 2). JSHint is good for catching this sort of thing.